modules/integration/tests-common/common-artifacts/pom.xml

```diff
@@ -43,6 +43,8 @@
         <module>jaxrs</module>
         <module>security-check</module>
         <module>logging</module>
+        <module>carboncontext-test-app</module>
+        <module>cache-test-app</module>
```

Comment:
since this in the integration module, do we need to name the app with "test-app"?